pandapower/create.py

```diff
@@ -3581,17 +3582,29 @@ def create_impedance(net, from_bus, to_bus, rft_pu, xft_pu, sn_mva, rtf_pu=None,
         rtf_pu = rft_pu
     if xtf_pu is None:
         xtf_pu = xft_pu
+    if rft0_pu is not None and rtf0_pu is None:
+        rtf0_pu = rft0_pu
```

Comment:
I'm not sure if it is necessary - but what if rtf0_pu is not None and rft0_pu is None?

Reply:
thank you I will add an error raise